spec/integration/v2/jobs_spec.rb

```diff
@@ -38,11 +71,15 @@
   end
 
   context 'GET /jobs/:job_id/log.txt' do
-    it 'returns log for a job' do
-      stub_request(:get, "#{Travis.config.logs_api.url}/logs/#{job.id}?by=job_id&source=api")
-        .to_return(status: 200, body: JSON.dump(content: 'the log'))
+    it 'returns redirects to archived log url' do
+      response = get("/jobs/#{job.id}/log.txt")
+      expect(response.status).to eq(307)
+      expect(response.location).to eq(archived_log_url)
+    end
+
+    it 'returns 406 (Unprocessable) if Accept header requests JSON' do
```

Comment:
This is a new (and correct) behavior, where the request for a `.txt` content with `Accept: application/json` is rejected as unprocessable.